code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -82,52 +109,60 @@
 			if(slot_item && !(slot_item.type in chronosafe_items) && user.unEquip(slot_item))
 				user << "<span class='notice'>Your [slot_item.name] got left behind.</span>"
 
-		phaseanim.name = "phasing [user.name]"
-		phaseanim.icon = 'icons/mob/mob.dmi'
-		phaseanim.icon_state = "chronostuck"
-		phaseanim.density = 1
-		phaseanim.layer = FLY_LAYER
-		phaseanim.master = user
 		user.ExtinguishMob()
 		if(user.buckled)
 			user.buckled.unbuckle_mob()
-		user.loc = holder
-		flick("chronophase", phaseanim)
-		spawn(7)
-			if(user)
-				if(phaseanim)
-					if(camera && camera.loc)
-						to_turf = camera.loc
-						flick("chronounphase", phaseanim)
-					else
-						flick("chronostuck", phaseanim)
-					phaseanim.loc = to_turf
-					sleep(7)
-			if(holder)
-				if(user && user in holder.contents)
+
+		var/icon/user_icon = getFlatIcon(user)
+		user_icon.Blend("#ffffff")
+		phase_underlay = new(user.loc)
+		phase_underlay.icon = user_icon
+		phase_underlay.density = 1
+		phase_underlay.anchored = 1
+		phase_underlay.master = user
+		phase_underlay.animate_movement = NO_STEPS
+		phase_underlay.alpha = 0
+		phase_underlay.mouse_opacity = 0
+		phase_underlay.name = user.name
+		phase_underlay.transform = user.transform
+		phase_underlay.pixel_x = user.pixel_x
+		phase_underlay.pixel_y = user.pixel_y
+
+		user.animate_movement = NO_STEPS
```

Comment:
making it so you don't slide if you teleport to an adjacent tile, it does do exactly what I think it does